build.sbt

```diff
@@ -8,24 +8,25 @@ val Nightly = sys.env.get("TRAVIS_EVENT_TYPE").contains("cron")
 
 val Scala212 = "2.12.10"
 val Scala213 = "2.13.1"
-val akkaVersion26 = "2.6.6"
-val akkaVersion = if (Nightly) akkaVersion26 else "2.5.31"
-val AkkaBinaryVersion25 = "2.5"
-val AkkaBinaryVersion26 = "2.6"
-val AkkaBinaryVersion = if (Nightly) AkkaBinaryVersion26 else AkkaBinaryVersion25
+
+val AkkaBinaryVersionForDocs = "2.6"
+val KafkaVersionForDocs = "24"
+
+val akkaVersion = "2.6.9"
```

Comment:
Is `2.6.9` required? Could we use `2.6.6` ~or an older version~, for example?

(edited: it seems to me that we need at least 2.6.6)

Reply:
I upgraded to the latest because this PR will be part of a new minor release of Alpakka Kafka.